.vscode

```diff
@@ -0,0 +1 @@
+contrib/.vscode/
```

Comment:
Please remove this symlink, people can add it manually if they desire so

Reply:
Fixed in [d732d4e](https://github.com/d120/inforz/pull/1/commits/d732d4e4e2436c7ec434092d098348391f17e0f3)